q2_types/feature_data/tests/test_transformer.py

```diff
@@ -556,6 +606,55 @@ def test_skbio_tabular_msa_to_aligned_dna_fasta_format(self):
         for act, exp in zip(obs, input):
             self.assertEqual(act, exp)
 
+    def test_dnafasta_format_to_series(self):
+        filename = 'dna-sequences.fasta'
+        _, obs = self.transform_format(DNAFASTAFormat, pd.Series, filename)
+
+        seqs_fp = self.get_data_path(filename)
+        exp = skbio.read(seqs_fp, format='fasta', constructor=skbio.DNA)
+
+        for expected, observed in zip(exp, obs):
```

Comment:
This test case is a little too circular for comfort -- can you create an expected `Series` object to compare to?